docs/migration_guide.rst

```diff
@@ -20,6 +20,8 @@ Deprecation
 - ``compute_invoke_hash`` is deprecated in favour of ``compute_transaction_hash``
 - ``starknet_py.common.create_contract_class`` is deprecated in favour of ``starknet_py.common.create_compiled_contract``
 
+- Removed deprecated ``max_steps`` in :class:`~starknet_py.proxy.contract_abi_resolver.ProxyConfig`.
```

Comment:
I think it technically should go to "Breaking changes".

Reply:
Probably, but the breaking changes section is really long this version and this removal still touches the deprecation subject

Reply:
I wouldn't hide the change just because there are already a lot of other breaking changes. It's not the place to address this.